### ISSUE TITLE
failing SDRGES3 test for OpenBLAS 0.3.23 with GCC/12.3.0 on `aarch64` (Graviton 2)

### ISSUE THREAD

**Issue by boegel** [easybuilders/easybuild-easyconfigs#18017]
cfr. https://github.com/easybuilders/easybuild-easyconfigs/pull/18012#issuecomment-1574982631

**Comment by bartoldeman** [comment]
Does this patch fix it on aarch64? It's similar to what fixes it on Haswell, except for single precision instead of double precision.

```
--- OpenBLAS-0.3.23/lapack-netlib/TESTING/sgd.in.orig   2023-06-06 11:01:50.512947527 +0000
+++ OpenBLAS-0.3.23/lapack-netlib/TESTING/sgd.in        2023-06-06 11:02:05.318078733 +0000
@@ -1,6 +1,6 @@
 SGS               Data for the Real Nonsymmetric Schur Form Driver
 5                 Number of matrix dimensions
-2 6 10 12 20 30   Matrix dimensions
+6 2 10 12 20 30   Matrix dimensions
 1 1 1 2 1         Parameters NB, NBMIN, NXOVER, NS, NBCOL
 10                Threshold for test ratios
 .TRUE.            Put T to test the error exits
```

**Comment by bartoldeman** [comment]
Fixed in 0a838b38e5c2f10d9c1b4a15f5f8918e597d72d9

**Comment by Flamefire** [comment]
It looks like we have to reopen this issue as with  https://github.com/easybuilders/easybuild-easyconfigs/commit/0a838b38e5c2f10d9c1b4a15f5f8918e597d72d9  the exact same issue happens on PPC now
